### PR TITLE
English code issue

### PR DESCRIPTION
**Pull request by vladeugeniu**

### CODE REVIEW

File: python/fazan/fazan.py [hunk 1]
```diff
@@ -1,136 +1,111 @@
...
-        print "Ai fost incuiat, nu exista niciun cuvant care incepe cu", pref
-        return False
-    lines = []
-    f.seek(0)
-    for line in f:
-        lines.append(line)
+        self.answer = raw_input("raspunsul tau: ")
+        return self.answer
 
+    def add_faz(self):
```
Comment (razvand) [inline python/fazan/fazan.py:20]: Bad function name "add_faz". It tells me little about what the function does.

File: python/fazan/fazan.py [hunk 2]
```diff
@@ -1,136 +1,111 @@
...
-    print "Prefixul pentru cuvantul tau este: ", pref
-    i = 0
+        if self.score == 1:
+            self.fazan = self.fazan + 'f'
 
-    while i < 3:
-        alegere = player.my_turn();
-        alegere = alegere + '\n'
+        elif self.score == 2:
+            self.fazan = self.fazan + 'a'
```
Comment (razvand) [inline python/fazan/fazan.py:26]: Use something else in place of `fazan`.

File: python/fazan/fazan.py [hunk 3]
```diff
@@ -1,136 +1,111 @@
...
 
-    alegere = computer.my_turn(pref)
-    return alegere
 
+def computer_turn(pref, computer):
+
+    if pref in word_list:
+        choice = computer.my_turn(pref)
+        return choice
+    print "Felicitari, m-ai inchis! "
```
Comment (razvand) [inline python/fazan/fazan.py:85]: Remove extra blank before quotes sign.

File: python/fazan/fazan.py [hunk 4]
```diff
@@ -1,136 +1,111 @@
...
-    print computer.nume + ':', computer.fazan
+        print 'Cuvantul curent:', current_word
+        current_word = player_turn(current_word[len(current_word)-2:len(current_word)], player)  # -2 since there is no '\n' i have to send just the last 2 characters
+        if current_word is not False:
+                current_word = computer_turn(current_word[len(current_word)-2:len(current_word)], computer)  # same thing as abobe, sending just 2 characters
+                if current_word is False:
+                    computer.add_faz()
+                    break
+                else:
+                    print "answerul calculatorului: ", current_word
```
Comment (razvand) [inline python/fazan/fazan.py:105]: You should use "Raspunsul", not "anwserul" :-)

Comment (vladeugeniu) [inline python/fazan/fazan.py:105]: oh, aici am dat  replace all pentru o variabila, stiu, bad practice.

**Pull request merged**